src/frontend/src/btc/components/convert/BtcConvertProgress.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+	import type { ProgressStep } from '@dfinity/gix-components';
+	import InProgressWizard from '$lib/components/ui/InProgressWizard.svelte';
+	import { ProgressStepsConvert } from '$lib/enums/progress-steps';
+	import { i18n } from '$lib/stores/i18n.store';
+
+	export let convertProgressStep: string = ProgressStepsConvert.INITIALIZATION;
+
+	let steps: [ProgressStep, ...ProgressStep[]];
```

Comment:
@AntonioVentilii somewhere there should be a type for this now right?

Reply:
Just searched through the codebase and I found this - https://github.com/dfinity/oisy-wallet/blob/main/src/frontend/src/lib/types/transaction.ts#L44, which is not really general. So, maybe it's still only planned to create some kind of `NonEmptyArray<T>`?

Reply:
Sure. Already approved hours ago ;)